ballerina/data_mappings.bal

```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Same

Reply:
updated in [6df510d](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/6df510d1098d680d7b11812ca4ebd7b34b2af897)